.circleci/config.yml

```diff
@@ -49,6 +49,7 @@ parameters:
 orbs:
   gravitee: gravitee-io/gravitee@1.0
   secrethub: secrethub/cli@1.1.0
+  slack: circleci/slack@4.2.1
```

Comment:
I don't understand why this file contains some changes. Could you override it using the version available on master branch ?